### PR TITLE
Replacing service component with utils and misc in introduction page

### PR DESCRIPTION
**Pull request by napetrov**
In this change we reflect design decisions on intro page

### CODE REVIEW

File: source/elements/oneDAL/source/introduction.rst [hunk 1]
```diff
@@ -45,9 +45,14 @@ communication technology and, therefore, can be used within different end-to-end
    
    - Prediction algorithms typically work with the trained data model and with a working data set.
 
- - The **Services** component includes classes and utilities used across Data Management and Algorithms components.
-   These classes enable memory allocation, error handling.
+ - The **Utilities**(util) component includes auxiliary functionality intended to be used for design of
+ classes and implementation of methods such as memory allocators or type traits.
```
Comment (outoftardis) [inline source/elements/oneDAL/source/introduction.rst:49]: please align with #144 

```
auxiliary functionality, such as memory allocators or type traits, that is intended to be used for the design of classes and implementation of various methods.
```

File: source/elements/oneDAL/source/introduction.rst [hunk 2]
```diff
@@ -45,9 +45,14 @@ communication technology and, therefore, can be used within different end-to-end
...
 
-Classes in Data Management, Algorithms, and Services components cover the most important usage scenarios and allow
-seamless implementation of complex data analytics workflows through direct API calls. At the same time, the library
-is an object-oriented framework that helps customize the API by redefining particular classes and methods of the library.
+ - The **Miscellaneous**(misc) component includes functionality intended to be used by |dal_short_name|
+ algorithms and applications for algorithm customization and optimization on various stages of the
+ analytical pipeline. Examples of such algorithms includes solvers, random number generators and others.
```
Comment (outoftardis) [inline source/elements/oneDAL/source/introduction.rst:53]:
```suggestion
 analytical pipeline. Examples of such algorithms include solvers and random number generators.
```

File: source/elements/oneDAL/source/introduction.rst [hunk 3]
```diff
@@ -45,9 +45,14 @@ communication technology and, therefore, can be used within different end-to-end
    
    - Prediction algorithms typically work with the trained data model and with a working data set.
 
- - The **Services** component includes classes and utilities used across Data Management and Algorithms components.
-   These classes enable memory allocation, error handling.
+ - The **Utilities**(util) component includes auxiliary functionality intended to be used for design of
```
Comment (andreynv) [inline source/elements/oneDAL/source/introduction.rst:48]: I'd remove util, misc terms

Comment (napetrov) [reply source/elements/oneDAL/source/introduction.rst]: done

**Pull request merged**